Tests/PointFreeTests/__Snapshots__/StripeWebhooksTests/testValidHook.1.Conn.txt

```diff
@@ -6,7 +6,7 @@
   Cookie: pf_session={}
   Stripe-Signature: t=1517356800,v1=499156b6abcf65d5c4a7c31f4e367d788b6112a030106c93aa2fc9fb1023473e
 
-  {"id":"evt_test","data":{"object":{"id":"in_test","amount_paid":1700,"closed":true,"lines":{"has_more":false,"data":[{"amount":1700,"id":"ii_test","quantity":1,"subscription":"sub_test","plan":{"amount":1700,"name":"Individual Monthly","id":"individual-monthly","created":1517356800,"currency":"usd","metadata":{},"interval":"month"}}]},"customer":"cus_test","charge":{"amount":1700,"id":"ch_test","source":{"exp_year":2020,"brand":"Visa","customer":"cus_test","id":"card_test","exp_month":1,"last4":"4242"}},"total":1700,"period_start":1517356800,"amount_remaining":0,"subtotal":1700,"date":1517356800,"period_end":1519948800,"number":"0000000-0000","subscription":"sub_test"}},"type":"invoice.payment_failed"}
+  {"id":"evt_test","data":{"object":{"id":"in_test","amount_paid":1700,"closed":true,"lines":{"has_more":false,"data":[{"amount":1700,"id":"ii_test","quantity":1,"subscription":"sub_test","plan":{"amount":1700,"name":"Individual Monthly","id":"individual-monthly","created":1517356800,"currency":"usd","metadata":{},"interval":"month"}}]},"customer":"cus_test","charge":"ch_test","total":1700,"period_start":1517356800,"amount_remaining":0,"subtotal":1700,"date":1517356800,"period_end":1519948800,"number":"0000000-0000","subscription":"sub_test"}},"type":"invoice.payment_failed"}
 
 ▿ Response
-  Status 200 OK
\ No newline at end of file
+  Status 400 BAD REQUEST
```

Comment:
Fixing these tests now. Need to update the sigs.

Reply:
so we expect a 400 on this?